### ISSUE TITLE
Office 365 no longer accepts "Pa55w.rd"

### ISSUE THREAD

**Issue by itbland** [MicrosoftLearning/20347-Enabling-and-Managing-Office365#63]
"Password is too common" error when trying to change/update passwords in Mod 02

**Comment by JasonHershey** [comment]
Great catch.  Discussed this with the team developing another course and we've confirmed that all of the common passwords we've used in courses are a problem when setting or changing passwords in O365. For future courses, the plan is to tell students to create a password, write it down, and keep using it.  Its a bit more unwieldy, but really the only solution that will work.  I'll go through and update Mod 2, and any instructions in other mods that reference the password.

**Comment by JasonHershey** [comment]
I've updated the labs to reflect this issue. In Module 1, student creates the password they will use for Office 365 accounts and writes it down.  In later instructions to create or enter a password, the instructions refer to the password they created in Module 1.  Post lab 4, there are mixed instructions as accounts synced with ADFS should have the on-premise password.    There are also a couple of places where in later modules the online password is changed so students must write THAT password down and use it.    There is potential that I incorrectly identified an o365 account that is 'cloud only' vs synced and didn't change the password, but I think I got them all.
